IRremoteESP8266.cpp

```diff
@@ -800,6 +797,13 @@ bool IRrecv::decode(decode_results *results) {
     return false;
   }
 
+  for (int i = 0; i < irparams.rawlen; i++) {
+    if (irparams.rawbuf[i] < 0)
+      irparams.rawbuf[i] = (0xFFFFFFFF + irparams.rawbuf[i]) / USECPERTICK + 1;
```

Comment:
See previous comments, but just to note 0xFFFF is the max for a uint16/unsigned int on this platform. 0xFFFFFFFF was used because we were dealing with uint32s for the time calcs and just storing the deltas in rawbuf.